scripts/lb-wrapper

```diff
@@ -6,6 +6,7 @@ URL_OR_SEARCH_TERM="$2"
 LOG_FILE="/var/log/xklb.log"
 XKLB_DB_FILE="/library/calibre-web/xklb-metadata.db"
 TMP_DOWNLOADS_DIR="/library/downloads/calibre-web"
+OUTTMPL="'${TMP_DOWNLOADS_DIR}/%(extractor_key,extractor)s/%(uploader,uploader_id)s/%(title).170B_%(view_count)3.2D_[%(id).64B].%(ext)s'"
```

Comment:
Is there xklb documentation for this syntax specifying PATH/FILENAME ?

(With an illustrative example ideally?!)

Reply:
Preference: An explanatory comment above Line 9 would be incredibly useful 😇

Reply:
Please see https://github.com/yt-dlp/yt-dlp?tab=readme-ov-file#output-template